Src/AutoFakeItEasy/MethodCallResult.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+
+namespace AutoFixture.AutoFakeItEasy
+{
+    internal class MethodCallResult
+    {
+        private readonly IList<PositionedValue> outAndRefValues;
```

Comment:
Probably that's a premature optimization, but I'd initialize it lazily, as in 99% it's not used. Don't use `Lazy<T>` - it's the same allocation 😉